index.html

```diff
@@ -59,13 +59,10 @@ <h1 class='title'>hugo findlay's web site</h1>
         top: 0px;
         z-index: -1;
         user-select: none;
-        /* 
-        TODO day mode
-
-        background-color: #fff0d1;
-        color: #2d2821; 
-        
-        */
+        @media (prefers-color-scheme: light) {
+            background-color: #fff0d1;
+            color: #2d2821;
+        }
```

Comment:
could also use [light-dark](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark) to avoid media selector